substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/UnalignedHeapChunk.java

```diff
@@ -202,6 +202,18 @@ public static void dirtyCardForObjectOfUnalignedHeapChunk(Object obj, boolean ve
         }
     }
 
+    /**
+     * Dirty the card corresponding to the given Object without verify.
+     *
+     * This has to be fast, because it is used by the post-write barrier.
+     */
+    public static void dirtyCardForObjectOfUnalignedHeapChunk(Object obj) {
```

Comment:
Here not calling `dirtyCardForObjectOfUnalignedHeapChunk(obj, false)` is because that method has the verify code: `AssertionNode.assertion(false, CardTable.isDirtyEntryAtIndexUnchecked...` , and `AssertionNode` will be called at compile time.
If we call this at GC will get exception here, could I just change this AssertionNode or implement a no verify one